src/actions/helpers/helpers.js

```diff
@@ -133,11 +150,23 @@ function mapPeriodtoDate(period, moment) {
   throw new customError('incorrect period', EA1006)
 }
 
+/**
+ * Maps a string that contains a number of days to a number e.g) '2 days' to 2
+ * @param {String} zone
+ * @param {moment-timezone} moment_tz
+ */
+function timezoneExists(zone, moment_tz) {
```

Comment:
moment_tz is passed around to get here. Can you think of a better way to inject this dependency, instead of explicitly passing it around?